cmd/packageExport.go

```diff
@@ -76,12 +82,21 @@ func newPackageExportCommand() *packageExportCommand {
 	cmd.Flags().BoolVarP(&template, "template", "t", false, "Export a package config template")
 	cmd.Flags().BoolVarP(&exportAll, "all", "a", false, "Export all packages")
 	cmd.Flags().StringVarP(&destination, "destination", "d", ".", "Destination for the export")
+	cmd.Flags().BoolVarP(&stdout, "stdout", "o", false, "Export package(s) to stdout")
 
 	_ = cmd.MarkFlagDirname("destination")
 
 	return &packageExportCommand{cmd: cmd}
 }
 
+func exportPackage(pkg *domain.Package, destination *string) (*string, error) {
+	if destination != nil {
+		exportedTo, err := session.packageService.ExportPackageToConfig(*pkg, *destination)
+		return &exportedTo, err
+	}
+	return nil, session.packageService.ExportPackageToStdout(*pkg)
+}
+
```

Comment:
The `exportPackage` function should look something like this after the above suggestions have been implemented:

```go
func exportPackage(pkg *domain.Package, destination string, toStdout bool) (string, error) {
	if toStdout {
		return "stdout", session.packageService.ExportPackageToStdout(*pkg)
	}

	return session.packageService.ExportPackageToConfig(*pkg, destination)
}
```